CHANGES.rst

```diff
@@ -25,6 +25,33 @@ Build:
 Test:
  -
 
+Changes to Matrix Android SDK in 0.9.16 (2018-12-20)
+=======================================================
+
+Features:
+ -
+
+Improvements:
+ - isValidRecoveryKey() ignores now all whitespace characters, not only spaces
```

Comment:
It's not necessary to add a new block of changes, the current block at the top of file is already here to add changes for the next release. So you just need to add two lines for you PR (this one and the one in the Test section).
Anyway I will fix this after merging your PR.